tests/src/test/scala/com/lightbend/paradox/markdown/SnipDirectiveSpec.scala

```diff
@@ -89,7 +89,7 @@ class SnipDirectiveSpec extends MarkdownBaseSpec {
       |<pre class="prettyprint">
       |<code class="language-scala">
       |case object Dent
-      |case object DoubleDent</code>
+      |  case object DoubleDent</code>
```

Comment:
this one seems suprising?

Reply:
That's what Akka asked for...
The example is for how Alpakka wanted it to be.